redhat-developer/app-services-cli
# feat(context): add commands for context management

Add commands to create and manage service contexts.

### Verification Steps
1. Create a new service context.
```
rhoas context create
```
2. Verify the service commands use the services specified in the context. (currently done only for `rhoas service-registry rule` commands).
```
rhoas service-registry rule list
```
3. Verify other context commands - `use`, `list`, `status`, `delete`.

### To Do

1. Handle migration of services from config file to context file.
2. Status should display status of services (instead of displaying them).
3. Switch from `config` to `context` in rest of the commands.
4. Updating current context using `kafka use` and `service-registry use`

### Type of change


### Checklist
